p2p/dialer.go

```diff
@@ -174,6 +187,16 @@ func (m *dialer) exchangeConfig(consumerID, providerID identity.Identity, servic
 		return nil, fmt.Errorf("could not send signed msg: %v", err)
 	}
 
+	// TODO: put this under feature set check when its available
+	// wait until provider confirms that channel handlers are ready
+	select {
```

Comment:
We don't have any provider yet with this new feature. I think this TODO can be removed.